whichone/static/js/data.js

```diff
@@ -107,18 +72,42 @@ async function request(key) {
   });
 }
 
-async function getSpotifyData() {
-  request("top_artists").catch(() => {
-    console.error("API unavailable. Please try again later.");
+function processAudioFeatures(audio_features) {
+  let output = {};
+  audio_features.forEach((result) => {
+    output[result["id"]] = result;
   });
+  return output;
```

Comment:
Consider [reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce) to clean this up